inttests/sds_test.go

```diff
@@ -124,23 +125,35 @@ func TestCreateSdsInvalid(t *testing.T) {
 }
 
 // TestCreateSdsInvalid will attempt to add an SDS, which results in failure
-func TestCreateSdsIPRoleInvalid(t *testing.T) {
+func TestCreateSdsParamsInvalid(t *testing.T) {
 	pd := getProtectionDomain(t)
 	assert.NotNil(t, pd)
 
 	// attempt to create an SDS with a number of invalid IPs
 	// this is done, in a failure mode, to prevent changing the Protection Domain used for testing
 	sdsName := "invalid"
```

Comment:
Hi @rounak-adhikary, can we have some mock values that tests against valid test cases as well.
Also applicable to other tests methods where only invalid scenarios are tested.

Reply:
@khareRajshree For valid testcases for SDS we need atleast one real IP. Is there a dedicated IP that I can use? I can reuse this testcase (https://github.com/dell/goscaleio/blob/sds-tf-test/inttests/sds_test.go#L149) that I was using.

Reply:
I believe that the lack of dedicated IPs is what led to the existence of only invalid testcases for SDS.

Reply:
yea that's also true, we can leave the valid TC for now, as we cannot put the internal systems IPs out here.